docs/podman-rm.1.md

```diff
@@ -1,9 +1,11 @@
 % podman-rm(1)
 
 ## NAME
-podman\-rm - Remove one or more containers
+podman\-container\-rm (podman\-rm) - Remove one or more containers
```

Comment:
same comment here as in the podman rmi page.  If this is the podman rm page, `podman rm` should be listed first.

Reply:
Well it is both a podman-rm and a podman-container-rm man page.  The name on the file means nothing.  Perhaps we should change the file name.  
podman-container-rm is more descriptive of what the command does, so I believe should be done first and the second is really more of an alias.  (Although historically they are reversed.)